lib/karafka/testing/minitest/proxy.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Karafka
+  module Testing
+    module Minitest
```

Comment:
I believe it is called `MiniTest` and not `Minitest`

Reply:
I adjusted